Dockerfile

```diff
@@ -1,3 +1,17 @@
+# Generate static assets (CSS and JavaScript)
+FROM node:18.16 AS staticassets
+WORKDIR /home
+COPY package.json package-lock.json webpack.config.js ./
+RUN npm install
+COPY scripts ./scripts
+COPY sass ./sass
+RUN npx webpack --config webpack.config.js
```

Comment:
Hi @ahosgood Was excited to give this a go, but things are hanging here for me when rebuilding locally (gave up after 5 mins). Any ideas?

Reply:
No idea - is this the step it's hanging on (`npx webpack`)?

Reply:
Yep!

Reply:
That's really strange. This was meant to be the first part of getting a Docker image that could be deployed on AWS so building the assets needed to happen at this stage...

The command is run inside a container so it shouldn't be a difference in OS or environment which makes me feel like it could be more of a network issue?

Would you be able to try it by replacing the command with:

```sh
RUN npx webpack --verbose --config webpack.config.js
```

...and then share where it gets to on your machine?

Reply:
Latest version is still hanging for me, unfortunately. Maybe something to do with me being on an M1 Macbook Pro?

Reply:
Hmm... quite possibly. I'm on an Apple M2 Pro chip - I wonder if there's a difference.

Will investigate!

Reply:
@ababic - I've now switched this to use the npm scripts which are no precompiled like npx scripts. Hopefully that means this step will rely on the `sass` library that gets installed from the `package.json`.

Really sorry to ask, but could you please give this another whirl?